src/OmniSharp.Roslyn.CSharp/Services/Buffer/ChangeBufferService.cs

```diff
@@ -0,0 +1,26 @@
+using System.Composition;
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Buffer
+{
+    [OmniSharpHandler(typeof(RequestHandler<ChangeBufferRequest, object>), LanguageNames.CSharp)]
```

Comment:
We could, but then we'd lose some of the runtime analysis that Mef can give us, since it validates that the ContractType (the handler) matches the type that the attribute is attached to.